components/Faq.js

```diff
@@ -1,7 +1,149 @@
 import React from "react";
+import Script from "next/script";
+
+let colorCount = 0;
+
+const bodyColors = [
+	{
+		backgroundColor: "#82aaff",
+	},
+	{
+		backgroundColor: "#c792ea",
+	},
+	{
+		backgroundColor: "#ffd700",
+	},
+	{
+		backgroundColor: "#c3e88d",
+	},
+	{
+		backgroundColor: "#ff5370",
+	},
+	{
+		backgroundColor: "#f07178",
+	},
+	{
+		backgroundColor: "#89ddff",
+	},
+	{
+		backgroundColor: "#f78c6c",
+	},
+	{
+		backgroundColor: "#80cbc4",
+	},
+	{
+		backgroundColor: "#da70d6",
+	},
+];
+
+const infoArray = [
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+];
+
+const FaqItem = props => {
+	if (colorCount == bodyColors.length) {
+		colorCount = 0;
+		console.log("Color reset to 0!");
+	}
+
+	return (
+		<>
+			<div className='accordion-item'>
```

Comment:
Use React Bootstrap instead of vanilla Bootstrap, it will give you components instead and are much easier to use